api/users/serializers.py

```diff
@@ -596,7 +603,7 @@ def create(self, validated_data):
             token = user.add_unconfirmed_email(address)
             user.save()
             if CONFIRM_REGISTRATIONS_BY_EMAIL:
-                send_confirm_email(user, email=address)
+                enqueue_postcommit_task(send_confirmation_email_task, user.id, address)
```

Comment:
hmm, this is in the API serializer, ORCiD SSO doesn't use API at all, it's some legacy code under `/framework/auth` IIRC

Reply:
yep, it's here https://github.com/CenterForOpenScience/osf.io/blob/develop/framework/auth/views.py#L804

in addition, this `send_confirm_email()` is used by many legacy auth views, may need to touch things outside the scope of the ticket